frontend/src/hooks/useFileDownload.tsx

```diff
@@ -0,0 +1,13 @@
+import { useCallback } from "react";
+
+export const useFileDownload = () => {
```

Comment:
We have a lib for this called `file-saver` something like this. This is because file downloading is a bit complex due to browser compatibility

Reply:
Ah this thing is not used at all anymore, it was a leftover from the removal of recovery